### PR TITLE
WIP: updating docs for CNCF submission

### PR DESCRIPTION
**Pull request by endophage**
- [x] removed some docker specific content as we're not using the contents of `./docs/` to build official docker docs any more.
- [x] Add comparative analysis to README
- [x] Add use cases/scope statement to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +53,38 @@ Our last security audit was on July 31, 2015 by NCC ([results](docs/resources/nc
...
+Traditional Package Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+rkt trust | ✔ | ❌  | ❌  | ❌  | ❌  
+Red Hat Container Image Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+YUM | ✔ | ❌  | ❌  | ❌  | ❌  
+APT | ✔ | ❌  | ❌  | ❌  | ❌  
+PyPI | ✔ | ❌  | ❌  | ❌  | ❌  
+GPG | ✔ | ❌  | ❌  | ❌  | ❌  
+No Signing | Partial (TLS) | ❌  | ❌  | ❌  | ❌  
+
+#### Protection from attacks
```
Comment (NathanMcCauley) [inline README.md]: This could use a section with a one-sentence description of each attack.

Comment (endophage) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -39,6 +53,52 @@ Our last security audit was on July 31, 2015 by NCC ([results](docs/resources/nc
...
+
+Definitions taken from the [TUF specification](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.md):
+
+- Arbitrary installation attacks. An attacker installs anything they want on the client system. That is, an attacker can provide arbitrary files in response to download requests and the files will not be detected as illegitimate.
```
Comment (riyazdf) [inline README.md]: could we bold the attack names and link the table column headers back to these definitions?

**Comment by HuKeping** [comment]
Still pending for something?

**Pull request closed without merging**